packages/http-smoke-testing/.travis.yml

```diff
@@ -21,4 +21,3 @@ script:
   - php vendor/bin/php-cs-fixer fix --config=vendor/shopsys/coding-standards/build/phpcs-fixer.php_cs --dry-run --verbose --diff ./src ./tests
   - php vendor/bin/phpcs --standard=vendor/shopsys/coding-standards/rulesetCS.xml --extensions=php --encoding=utf-8 --tab-width=4 -sp ./src ./tests
   - php vendor/bin/phpmd "./src,./tests" text vendor/shopsys/coding-standards/rulesetMD.xml --extensions=php
-  - php vendor/bin/phpunit tests
```

Comment:
This change is not mentioned in the commit message. It doesn't belong here maybe

Reply:
Thanks for noticing, I've managed to confuse http-smoke testing with migrations, as you can see in https://github.com/shopsys/shopsys/pull/96/commits/2fdcb5be82fe2766226484206716c6edd0d9e1d8

Reply:
I've fixed it